src/sqlfluff/core/dialects/dialect_snowflake.py

```diff
@@ -169,6 +170,22 @@ class StatementSegment(ansi_StatementSegment):
     )
 
 
+@snowflake_dialect.segment()
+class CommentSegment(BaseSegment):
```

Comment:
CreateStatementCommentSegment
 ?

Reply:
just because reading through the parser Grammar, a comment would naturally make me think of `-- some comment`

Reply:
Gotcha, that makes sense to me!